bin/main.rb

```diff
@@ -1,2 +1,53 @@
 #!/usr/bin/env ruby
-puts "Hello world!" 
\ No newline at end of file
+# frozen_string_literal: true
+
+puts 'You are welcome to this TIC-TAC-TOE game'
+puts 'Please state your name'
+Player1 = gets.chomp.capitalize
+puts "#{Player1},PLease pick your preferred letter,X or O"
+tool1 = gets.chomp.upcase
+until tool1=="X" || tool1=="O"
+  puts "Please,pick either 'X' or 'O'?"
+  tool1=gets.chomp.upcase
+end
+tool2 = tool1 == 'X' ? 'O' : 'X'
+puts 'Second player,please state your name'
+Player2 = gets.chomp.capitalize
+until Player2!=Player1
+  puts "That name has been taken,please pick another name"
+  Player2=gets.chomp.capitalize
+end
+puts ""
+puts "#{Player2},The first player chose '#{tool1}',so you are stuck with '#{tool2}' for this round"
+puts ''
+puts "Now that's out of the way,let the game begin"
+puts ''
+position1 = 10
+arr = []
+while (position1 < 1 || position1 > 9) && !arr.include?(position1)
+  puts "#{Player1}, What position would you like to make your move?, PLease pick a number between 0 and 10"
+  position1 = gets.chomp.to_i
+  until !arr.include? position1
```

Comment:
- [ ] The loop is here expecting for input for a given player, but its role should be looping the whole game until a winner is found. Example:


```
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end

```

so try to adjust to this, (only one loop should be enough to mock the game)